server/incident/service.go

```diff
@@ -674,6 +738,17 @@ func (s *ServiceImpl) UpdateStatus(incidentID, userID string, options StatusUpda
 		return err
 	}
 
+	if options.Status == StatusArchived {
+		if incidentToModify.WebhookOnArchiveURL != "" {
+			go func() {
+				if err = s.sendWebhookOnArchive(*incidentToModify); err != nil {
+					s.pluginAPI.Log.Warn("failed to send a POST request to the archive webhook URL", "webhook URL", incidentToModify.WebhookOnArchiveURL, "error", err)
+					_, _ = s.poster.PostMessage(incidentToModify.BroadcastChannelID, "Incident archived announcement through the outgoing webhook failed. Contact your System Admin for more information.")
```

Comment:
The ID of the incident's channel is stored in the `ChannelID` field. The `BroadcastChannelID` field contains the ID of a channel where the incident creation is optionally broadcasted. That's why we were not seeing the error in the incident's channel:
 
```suggestion
					_, _ = s.poster.PostMessage(incidentToModify.ChannelID, "Incident archived announcement through the outgoing webhook failed. Contact your System Admin for more information.")
```

Reply:
@agarciamontoro thanks for correction! I've pushed a new commit to fix that.